fontbe/src/error.rs

```diff
@@ -97,6 +97,8 @@ pub enum Error {
     CompositesStalled(Vec<GlyphName>),
     #[error("Inconsistent palette lengths observed: {0:?}")]
     InconsistentPaletteLength(Vec<usize>),
+    #[error("No metrics instance for '{0:?}'")]
+    NoGlobalMetricsInstance(NormalizedLocation),
```

Comment:
actually, the correct thing to do is _not_ to error when we don't have a GlobalMetricsInstance at the same location as a given glyph instance, but we should interpolate one! https://github.com/googlefonts/fontc/pull/1518 was added by @Hoolean exactly for this purpose.

Reply:
Should that just be how accessing metrics works? Is there a case where we *don't* want that?